stormpath/stormpath-sdk-react
# Add client API support

Adds support for the new client API feature.

#### How to verify

Using the [example application](https://github.com/stormpath/stormpath-express-react-example) test that the example application is running as previously (that you can login/logout, refresh the page and maintain the session, etc). Open up the network console and verify that all requests are called on the same domain and using regular cookie authentication.

Once you've verified that the example application works as previously, open up `app.js` in the and set the `endpoints.baseUri` option as shown below:

```javascript
ReactStormpath.init({
  endpoints: {
    baseUri: 'https://oval-summer.apps.dev.stormpath.io'
  }
});
```

Once that is done, redo the previous tests (that you can login/logout, refresh the page and maintain the session, etc), but this time in the network console, verify that all requests are against the `oval-summer.apps.dev.stormpath.io` domain and that no errors are present in console.

Fixes #130
